pyfar/classes/coordinates.py

```diff
@@ -1857,11 +1128,11 @@ def find_within(
                 points, distance + atol, return_sorted=return_sorted)
         if distance_measure in ['spherical_radians', 'spherical_meter']:
             # determine validate radius
-            radius = self.radius
+            radius = np.concatenate((self.radius, find.radius))
```

Comment:
Is this from another pull? I vaguely remember that this or something similar popped up somewhere else. There we used `.flatten()` for both sets of radii.

Reply:
main was not merged into dev, thats why sth went wrong. No it should work